membership-attribute-service/conf/application.conf

```diff
@@ -24,4 +24,22 @@ identity.prerelease-users = []
 
 play.filters.csrf.contentType.blackList = ["text/plain"]
 
+contexts {
+  jdbc-context {
+    executor = "thread-pool-executor"
+    throughput = 1
+    thread-pool-executor {
+      fixed-pool-size = 30
+    }
+  }
+}
+
+db.oneOffStore {
+  url=null
```

Comment:
do these come from S3? did we create a read-only user for the members-data-api?

Reply:
yes they'll be in the s3 config file. I'll make sure there's a new read-only user (have done so in CODE)
I've set them to null here for the sake of documentation, but open to other ideas.
The url will be that of the read replica